lib/ex_doc.ex

```diff
@@ -25,6 +25,7 @@ defmodule ExDoc do
       project: nil,
       retriever: ExDoc.Retriever,
       source_beam: nil,
```

Comment:
oh. I see now what you mean.
I was thinking why not have a default empty string.
I will be submitting another PR, and because we are using `nil` I need to check agains `nil`, `false` or `""`  to see if an option is unset. If by default we set to to empty, we just need to check against `false` or `""` (and false is because people may have just used `--foo` without value, and it because a boolean argv.